crates/event-streaming/build.rs

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Walrus Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{fs, path::PathBuf, process::Command};
+
+const SUI_REPO_URL: &str = "https://github.com/MystenLabs/sui.git";
```

Comment:
As mentioned I'll be moving all the proto files to another repo soon.

Reply:
will update the repo once you do it, will make the git clone much cheaper.